fw.lua

```diff
@@ -866,6 +866,12 @@ function _M.set_option(self, option, value)
 			local t = self._blacklist
 			self._blacklist[#t + 1] = value
 		end,
+		add_ruleset = function(value)
+			if (not _table_has_value(self, value, self._active_rulesets)) then
+				local t = self._active_rulesets
+				self._active_rulesets[#t + 1] = value
+			end
+		end,
```

Comment:
This logic will have the rulesets processed in the order they were added. This would also force any custom rulesets to be processed after all the included rulesets. Is this desired behavior?

Perhaps instead of (or in addition to) adding rulesets, we should provide a way to override the entire _active_rulesets table. Thoughts welcome!